doc/source/enhancements.rst

```diff
@@ -1,12 +1,333 @@
+as well. See :ref:`component_configuration` for more information.
+
+Enhancements can be defined in a ``generic.yaml`` file that is always loaded
+for all data or in an instrument-specific file (e.g. ``seviri.yaml``)
+corresponding to the ``.attrs["sensor"]`` metadata of the ``DataArray`` being
+processed. Generic enhancements are loaded first followed by sensor-specific
+enhancement files.
+
+Enhancement YAML Format
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The enhancement YAML format starts with an ``enhancements:`` name followed
+a series of enhancement "sections". An example file might look like:
+
+.. code-block:: yaml
+
+   enhancements:
+     default:
+       operations:
+       - name: stretch
+         method: !!python/name:satpy.enhancements.stretch
+         kwargs: {stretch: linear}
+     reflectance_default:
+       standard_name: toa_bidirectional_reflectance
+       operations:
+       - name: linear_stretch
+         method: !!python/name:satpy.enhancements.stretch
+         kwargs: {stretch: 'crude', min_stretch: 0.0, max_stretch: 100.}
+       - name: gamma
+         method: !!python/name:satpy.enhancements.gamma
+         kwargs: {gamma: 1.5}
+     overview:
+       standard_name: overview
+       operations:
+         - name: inverse
+           method: !!python/name:satpy.enhancements.invert
+           args: [False, False, True]
+         - name: stretch
+           method: !!python/name:satpy.enhancements.stretch
+           kwargs:
+             stretch: linear
+         - name: gamma
+           method: !!python/name:satpy.enhancements.gamma
+           kwargs:
+             gamma: [1.7, 1.7, 1.7]
+
+The enhancement section titles, in this case "default", "reflectance_default",
+and "overview", have no effect on the behavior of the enhancement and only
+serve as a unique identifier for the enhancement section. A section title
+must be unique or else one section will overwrite another. Order of the
+sections in a YAML file also has no effect.
+
+Inside an enhancement section are a series of enhancement "operations" which
+define, in order, the enhancement functions that should be applied to data
+matching this enhancement section. A function can be anything importable by
+Python (installed in your python environment or on your PYTHONPATH). Arguments
+and keyword arguments can be passed via ``args`` and ``kwargs``. The ``name``
+of the operation is primarily used for logging and does not need to be anything
+specific.
+
+The last part of the enhancement section are the matching terms to determine
+what data arrays should use this enhancement. See the below section for how
+enhancements are chosen for particular dataset.
+
+Real world examples can be found in Satpy source code directory such as
+``satpy/etc/enhancements/generic.yaml``.
+
+Matching Enhancements
+---------------------
+
+Choosing what enhancement should be applied to a particular ``DataArray``
+is based on the the metadata of that ``DataArray`'s ``.attrs``. Mapping
```

Comment:
Probably one missing closing backtick  for `DataArray`

Reply:
Good catch. I should check the final render and make sure this looks like I hope it does.

```suggestion
is based on the the metadata of that ``DataArray``'s ``.attrs``. Mapping
```